docs/2_4_common_co-simulation.adoc

```diff
@@ -245,22 +238,22 @@ _Therefore it is recommended that the FMU provides logging information to the us
 
 ===== Intermediate Variable Access [[api-intermediate-variable-access]]
 
-If the Co-Simulation master signals the support for intermediate variable access and an FMU has at least one variable with <<intermediateAccess>> = `true`, the FMU can invoke the <<fmi3CallbackIntermediateUpdate>> callback function to sent the <<fmi3IntermediateUpdateInfo>> back to the Co-Simulation master.
+If the Co-Simulation master signals the support for intermediate variable access and an FMU has at least one variable with <<intermediateAccess>> = `true`, the FMU can use the callback function <<fmi3CallbackIntermediateUpdate>> to communicate information back to the Co-Simulation master.
 
 [source, C]
 ----
-include::../headers/fmi3FunctionTypes.h[tag=IntermediateUpdateInfo]
+include::../headers/fmi3FunctionTypes.h[tag=CallbackIntermediateUpdate]
 ----
 
-The following flags in the `fmi3IntermediateInfo` struct are used for intermediate variable access:
+The following output arguments in <<fmi3CallbackIntermediateUpdate>> are used for intermediate variable access:
```

Comment:
The arguments are no output arguments of the function, but input arguments.
The word "output" should be deleted.